converter/base.py

```diff
@@ -114,7 +113,7 @@ def process_styles(fig_node: dict) -> Style:
     components = {}
     for inherit_style, copy_keys in SUPPORTED_INHERIT_STYLES.items():
         inherit_node_id = fig_node.get(inherit_style)
-        if not inherit_node_id or inherit_node_id[0] == 4294967295:
+        if inherit_node_id is None or utils.is_invalid_ref(inherit_node_id):
```

Comment:
I think `utils.is_invalid_ref` already checks if the id is None, right? So we can skip the check here.

Reply:
Indeed. I just need it for mypy compliance (it isn't smart enough to check that the function already does this check). I don't think it's worth adding a `# type: ignore` for this, so I kept the check both places, no harm done.

Reply:
:yells-at-mypy: